services/src/main/java/org/keycloak/protocol/oidc/endpoints/AuthorizationEndpointChecker.java

```diff
@@ -263,6 +267,32 @@ public void checkParRequired() throws AuthorizationCheckException {
         throw new AuthorizationCheckException(Response.Status.BAD_REQUEST, OAuthErrorException.INVALID_REQUEST, "Pushed Authorization Request is only allowed.");
     }
 
+    public void checkAuthorizationDetailsParam() throws AuthorizationCheckException {
+
+        if( request.getAuthorizationDetails() != null) {
```

Comment:
how about early exit to reduce nesting?

```java
if( request.getAuthorizationDetails() == null) {
  return;
}
```
...

Reply:
Yes, that is the better way, I will add.